README.md

```diff
@@ -80,7 +80,7 @@ Current status (not mentioned parts are not implemented):
 - [x] Atom - Unquoted
 - [x] Atom - Operator
 - [x] Atom - Quoted
-- [] Atom - Unicode support
+- [ ] Atom - Unicode support
```

Comment:
Possibly this should be `[x]`?

Reply:
Not yet, it would take a lot of regexes unless https://github.com/tree-sitter/tree-sitter/issues/95 gets resolved